lib/iris/experimental/um.py

```diff
@@ -685,19 +774,43 @@ def normalise(values, kind):
             output_file.seek((header.data_start - 1) * self._word_size)
             dataset_type = self.fixed_length_header.dataset_type
             sector_size = self._WORDS_PER_SECTOR * self._word_size
+
             for field in self.fields:
-                data = field.get_data()
-                if data is not None:
+                if isinstance(field, (Field2, Field3)):
+                    # Output 'recognised' lookup types (not blank entries).
                     field.lbegin = output_file.tell() / self._word_size
-                    # Round the data length up to the nearest whole
-                    # number of "sectors".
-                    size = data.size
-                    size -= size % -self._WORDS_PER_SECTOR
-                    field.lbnrec = size
-                    kind = {1: 'f', 2: 'i', 3: 'i'}.get(field.lbuser1,
-                                                        data.dtype.kind)
-                    data = normalise(data, kind)
-                    output_file.write(data)
+                    required_lbpack = field.lbpack
+                    field._check_valid_data_encoding(required_lbpack)
+                    if field._raw_data_is_useable_with_lbpack(required_lbpack):
```

Comment:
For now, just re-order, and consider renaming ``_raw_data_is_useable_with_lbpack`` to ``_raw_data_must_be_unpacked_to_save`` / ``_raw_data_must_be_processed_for_save`` or something of that ilk, to tell the reader that we are testing whether a pass through can take place. The ``useable`` term doesn't work for me - it is a little too generic.

The ``_check_valid is fine`` though. Kind of sounds like the work @mo-g is doing in #1635 though. I suspect it will change a little when packed data can be written.